core/contracts/oracle/implementation/Voting.sol

```diff
@@ -133,6 +132,14 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
 
     event VoteCommitted(address indexed voter, uint indexed roundId, bytes32 indexed identifier, uint time);
 
+    event EncryptedVote(
```

Comment:
This looks great, good optimization. My only nit is a comment here would be nice to explain how this functions to add to readability & understanding of this part of the code base. It's even more nuanced now that we emit an event.